sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/BlobsCheckpointStore.cs

```diff
@@ -31,7 +31,7 @@ protected BlobsCheckpointStore()
         public BlobsCheckpointStore(BlobContainerClient blobContainerClient,
             EventHubsRetryPolicy retryPolicy,
             string functionId,
-            ILogger logger): this(blobContainerClient, retryPolicy)
+            ILogger logger): this(blobContainerClient, retryPolicy, readLegacyCheckpoints: true)
```

Comment:
Should we consider renaming to something like `initializeWithLegacyCheckpoints` to reflect that the intent is to read them once and then replace them?

Reply:
Yep, will do.